### PR TITLE
[#9606] Fix course card arrows on Instructor Student Page

### LINKED ISSUE #9606: Improve students card in Instructor Student Page
- **Environment**: `master` branch at commit 9c97b9c3.

**Steps to reproduce**
1. Go to the `/web/instructor/students`page
1. Click on one of the courses

**Expected behaviour**
The DOWN arrow and UP arrow should be switched, i.e., show DOWN arrow when students are hidden, and UP arrow when the card is open.

**Actual behaviour**
The arrows do not correspond to the actions.

20190324_13h06_19

### PR DESCRIPTION
**Pull request by homps**
Fixes #9606 

**Outline of Solution**

Simple fix to swap the chevron classes on the course cards so down arrow is when expandable, up arrow is when collapsable.

Screenshot 2019-03-25 at 20 42 25

I've used snapshots to test each case. It's my first time using Jest snapshots, so use your eagle eyes on the tests :-)

Thanks for the opportunity with the firstTimers labels, it's a great idea to get people involved.

Also, I've got some feedback regarding the onboarding process. Let me know if you'd like me to do anything with it:
 1. I had Difficulties with installing the ngtest plugin in eclipse, from eclipse marketplace comments it looks like others have had a similar issue
 2. Ideally I'd have liked to run the ng tests in browser rather than headlessly (So I could debug). Couldn't easily see a config for that, Is that something I missed or isn't that currently an option?

### CODE REVIEW

**Comment by monmanuela** [comment]
> 1. I had Difficulties with installing the ngtest plugin in eclipse, from eclipse marketplace comments it looks like others have had a similar issue

I am not familiar with Eclipse so I can't advise on this. Any ideas or suggestions welcome :)

> 2. Ideally I'd have liked to run the ng tests in browser rather than headlessly (So I could debug). Couldn't easily see a config for that, Is that something I missed or isn't that currently an option?

Are you referring to debugging the snapshot tests? As far as I know, viewing the snapshot in browser is not an option, so it does require you to read through the snapshot with eagle eyes 👀

**Comment by wkurniawan07** [comment]
@homps, what kind of debugging are you looking for? I'm not sure whatever you wanted to achieve can be achieved with browser.

**Comment by homps** [comment]
> I am not familiar with Eclipse so I can't advise on this. Any ideas or suggestions welcome :)

It's probably more something to keep an eye out for in case others report the same headache :-)

> Are you referring to debugging the snapshot tests?
> @homps, what kind of debugging are you looking for? 

I was thinking something more like this: https://angular.io/guide/testing#test-debugging

Thanks both for your inputs.

**Comment by wkurniawan07** [comment]
@homps there is a way to do that,
```sh
node --inspect-brk node_modules/.bin/ng test --watch --runInBand
```
followed by adding `debugger;` line, which will act as breakpoint. But there has not been a pressing need for this yet.

**Comment by homps** [comment]
Thanks @wkurniawan07

File: src/web/app/pages-instructor/instructor-student-list-page/instructor-student-list-page.component.html [hunk 1]
```diff
@@ -138,8 +138,8 @@ <h2> Filter Students </h2>
                   [queryParams]="{courseid: course.id}">
             Enroll
           </button>
-          <i class="fas fa-chevron-down pl-3" [hidden]="!course.isChecked"></i>
-          <i class="fas fa-chevron-up pl-3" [hidden]="course.isChecked"></i>
+          <i class="fas fa-chevron-up pl-3" [hidden]="!course.isChecked"></i>
+          <i class="fas fa-chevron-down pl-3" [hidden]="course.isChecked"></i>
```
Comment (wkurniawan07) [inline src/web/app/pages-instructor/instructor-student-list-page/instructor-student-list-page.component.html]: These lines are all that you need to submit.

Comment (homps) [reply src/web/app/pages-instructor/instructor-student-list-page/instructor-student-list-page.component.html]: @wkurniawan07 no problem!
I've reverted the commit with the tests in.
Let me know if there's anything else I can do to help get this merged.

**Pull request merged**